crates/bevy_scene/src/dynamic_scene_builder.rs

```diff
@@ -348,9 +351,17 @@ impl<'w> DynamicSceneBuilder<'w> {
     /// [`deny_resource`]: Self::deny_resource
     #[must_use]
     pub fn extract_resources(mut self) -> Self {
+        let original_world_dqf_id = self
+            .original_world
+            .components()
+            .get_resource_id(TypeId::of::<DefaultQueryFilters>());
```

Comment:
It will be nice when resources-as-components lets us use default query filters to exclude the default query filters resource from our resource query :).